ton_client/src/debot/calltype.rs

```diff
@@ -40,6 +40,9 @@ struct Metadata {
     is_expire: bool,
     is_pubkey: bool,
     signing_box_handle: Option<SigningBoxHandle>,
+    ex_provided_timestamp: Option<u64>,
+    ex_provided_expire: Option<u32>,
+    ex_provided_pubkey: Option<String>,
```

Comment:
Let's rename this fields. Make it very simple like `timestamp`, `lifetime`, `pubkey`.